src/main/java/com/redhat/red/build/koji/model/xmlrpc/KojiArchiveInfo.java

```diff
@@ -42,6 +43,14 @@
     @JsonProperty( "id" )
     private Integer archiveId;
 
+    @DataKey( "btype" )
+    @JsonProperty( "btype" )
+    private String bType;
```

Comment:
Is it possible to make this field (and the associated one) a little more descriptive? I understand this is probably the field name in Koji, but it's ambiguous. Is it "binary type" or "build type" or something else? Making the field (and associated methods) a little more self-documenting would help avoid user confusion.

Reply:
My guess is that `bType` stands for *build type*, even though I only see it referenced as `btype` even if the API documentation. So, `buildType` may be more descriptive identifier to use. I will change it.

The type for builds themselves seems to be a list. I am not sure if this is the union of the build types of the archives or if its determined independently. In any case, this means that each archive only has a single build type, `String bType`, and that an archive's `bType` should be a subset of the build's types. So, these types should be the same as the ones in `KojiBuildTypeInfo`, but we're not currently using a `enum' or public constants for these values in the codebase.